packages/cli/src/bin/pipcook-daemon.ts

```diff
@@ -16,7 +16,8 @@ const BOOTSTRAP_HOME = path.join(DAEMON_HOME, 'bootstrap.js');
 interface DaemonBootstrapMessage {
   event: string;
   data: {
-    listen: number;
```

Comment:
We should keep it there if we are still using, or create a new interface `DaemonBootstrapV2`